src/substrate-indexed-data/orders/order.service.ts

```diff
@@ -236,21 +247,30 @@ export class OrderService {
       searchObj.size = _size;
     }
 
-    const total_orders = await this.elasticsearchService.count({
-      index: 'orders',
-      body: {
-        query: query,
-      },
-    });
+    let count = null
+    let data = []
 
-    const orders = await this.elasticsearchService.search(searchObj);
+    try {
+      const total_orders = await this.elasticsearchService.count({
+        index: 'orders',
+        body: {
+          query: query,
+        },
+      });
+  
+      const orders = await this.elasticsearchService.search(searchObj);
+      count = total_orders.body.count
+      data = orders.body.hits.hits
+    } catch (error) {
+      console.log('API "bounty_list/:customer_id":', error.body.error.reason);
```

Comment:
Instead of `console.log` use Loggers instead.

https://docs.nestjs.com/techniques/logger

Reply:
Either throw a new exception or return an `HttpException`. Don't return default values when error.